### ISSUE TITLE
force:org:shape:create returns a SOQL reference issue

### ISSUE THREAD

**Issue by ddawson235** [forcedotcom/cli#121]
### Summary

The `sfdx force:org:shape:create` command (when used in conjunction with a Shape enabled Dev Hub) returns the following error message:

```
ERROR running force:org:shape:create:  
SELECT SettingValue FROM orgPreferenceSettings
       ^
ERROR at Row:1:Column:8
No such column 'SettingValue' on entity 'OrgPreferenceSettings'. If you are attempting to use a custom field, be sure to append the '__c' after the custom field name. Please reference your WSDL or the describe call for the appropriate names.
```

### Steps To Reproduce:

1.  Ensure the DevHub configured in your CLI is Shapes enabled (pilot)
1. Run the command `sfdx force:org:shape:create -u ORGNAME` where `ORGNAME` is the org you wish to create a shape for.
1. The error above is returned from the request.

### Expected result

A new shape should have been created successfully and saved within the DevHub

### Actual result

```
ERROR running force:org:shape:delete:  
SELECT SettingValue FROM orgPreferenceSettings
       ^
ERROR at Row:1:Column:8
No such column 'SettingValue' on entity 'OrgPreferenceSettings'. If you are attempting to use a custom field, be sure to append the '__c' after the custom field name. Please reference your WSDL or the describe call for the appropriate names.
```

### Additional information

CLI version:
`sfdx-cli/7.11.2 darwin-x64 node-v8.11.2`

Operating system:
`macOS Mohave 10.14.5`

DevHub version:
`Summer 19`

**Comment by daveespo** [comment]
Thanks to Carlos Lopez over on the Org Shape Chatter group, you can work around the problem by downgrading the salesforcedx plugin

```
sfdx plugins:install salesforcedx@45.16.0
```

Then force:org:shape:create will work .. you can then return to the latest plugin by upgrading to salesforcedx@latest

**Comment by ddawson235** [comment]
granted this workaround unblocks users from creating a shape, but this issue highlights the need to fix the bug currently in version 46 👍

**Comment by rrmehta2** [comment]
@daviddawson90 Yes, this is a bug that has been prioritized by my team and will be fixed in the coming weeks. In addition to the workaround provided by @daveespo, you can also create a new shape by executing `sfdx force:data:record:create -u <shape org> -s ShapeRepresentation -v "description="`.  (in v46 of the CLI, no need to reinstall v45)

You can check the status of the shape using the org:shape:list and create a scratch org using shape as you normally do.

**Comment by balmasi** [comment]
> Thanks to Carlos Lopez over on the Org Shape Chatter group, you can work around the problem by downgrading the salesforcedx plugin
> 
> ```
> sfdx plugins:install salesforcedx@45.16.0
> ```
> 
> Then force:org:shape:create will work .. you can then return to the latest plugin by upgrading to salesforcedx@latest


I tried that but now it fails with a new error:

```
ERROR running force:org:shape:create:  Cannot create an AuthInfo instance that will overwrite existing auth data.

Try this:
Create the AuthInfo instance using existing auth data by just passing the username.  E.g., AuthInfo.create({ username: 'my@user.org' });
```

I'm definitely logged in cause my other commands work. Also I tried going back to latest and same thing.

In addition, running @rrmehta2 's solution yields:
`ERROR running force:data:record:create:  The requested resource does not exist`

**CLI Version**
`sfdx-cli/7.14.0 darwin-x64 node-v10.8.0`

**OS**
`MacOS Mojave 10.14`

**Devhub Version**
`Summer '19`

**Comment by rrmehta2** [comment]
@balmasi When you execute force:data:record:create you need to point to the org whose shape you want by either providing -u or assigning it to the defaultusername config. The "requested resource does not exist" error occurs if either of those aren't done. 

(Separately, we need to provide better error messages too).

**Comment by balmasi** [comment]
> @balmasi When you execute force:data:record:create you need to point to the org whose shape you want by either providing -u or assigning it to the defaultusername config. The "requested resource does not exist" error occurs if either of those aren't done.
> 
> (Separately, we need to provide better error messages too).

I'm aware of the -u flag. The problem persists with that flag. Does this have something to do with being a part of the pilot program? Is there something I need to check to see if I can use Org Shapes or not?

**Comment by rrmehta2** [comment]
@balmasi Yes, you will also see this error if you're not part of the Org Shape pilot and the feature has not been enabled on the specific org you're referencing with -u. You can check within the org's setup menu for a page called "Org Shape Subscribers" to confirm if the feature is enabled on your org.

**Comment by balmasi** [comment]
> @balmasi Yes, you will also see this error if you're not part of the Org Shape pilot and the feature has not been enabled on the specific org you're referencing with -u. You can check within the org's setup menu for a page called "Org Shape Subscribers" to confirm if the feature is enabled on your org.

Yea that was it, thanks for clearing that up
